ormar/queryset/queryset.py

```diff
@@ -924,6 +923,22 @@ async def first(self, *args: Any, **kwargs: Any) -> "T":
         self.check_single_result_rows_count(processed_rows)
         return processed_rows[0]  # type: ignore
 
+    async def first_or_none(self, *args: Any, **kwargs: Any) -> Optional["T"]:
+        """
+        Gets the first row from the db ordered by primary key column ascending.
+
+        If no match is found None will be returned.
+
+        :param kwargs: fields names and proper value types
```

Comment:
This as a result can also raise MultipleMatches, please add:
` :raises MultipleMatches: if more than 1 row is returned.`